code/modules/mob/living/simple_animal/hostile/hostile2.dm

```diff
@@ -169,6 +169,7 @@
 	LostTarget()
 
 /mob/living/simple_animal/hostile/proc/Goto(var/target, var/delay, var/minimum_distance)
+	step_towards(src, target) //weird but necessary so they try to bump openable obstacles
```

Comment:
theres a helper proc somewhere, get_step_towards2 that might address that. May be worth looking into.